src/components/AppBar/AppBar.tsx

```diff
@@ -138,9 +139,7 @@ const _AppBar = () => {
   const shareTitle = locationName ? locationShareTitle : defaultShareTitle;
 
   const trackShare = (target: string) => {
-    window.gtag('event', 'share', {
-      event_label: target,
-    });
+    trackEvent(EventCategory.ENGAGEMENT, EventAction.SHARE, target);
```

Comment:
Note: `engagement` is the default category for the `share` event, see https://developers.google.com/analytics/devguides/collection/gtagjs/events